conf/systemd/packetfence-proxysql.service

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) Inverse inc.
+[Unit]
+Description=PacketFence ProxySQL Load Balancer for connecting to clustered databases
+Before=packetfence-httpd.portal.service packetfence-api-frontend.service
+Wants=packetfence-config.service 
+Before=packetfence-docker-iptables.service
+Requires=packetfence-docker-iptables.service
+PartOf=packetfence-docker-iptables.service
+
+[Service]
```

Comment:
Does proxysql support sdnotify? If so we should definitely consider it since it's a critical service

Reply:
No support for sdnotify by default.